### PR TITLE
Camera and ML validation tests

### PR DESCRIPTION
**Pull request by Arthur-H21**
The fixed widths of the tables had to be slightly adjusted for the camera and ml tables

### CODE REVIEW

File: docs/design/3_1_1_video_capture.tex [hunk 1]
```diff
@@ -15,18 +15,18 @@ \subsubsection{MIPI Interface}
 The MIPI camera serial interface (CSI) is a high-speed protocol primarily intended for point-to-point image and video transmission between cameras and host devices. 
 
 %requirement got removed ...
-The MIPI protocol is a widely adopted protocol for image and video transmission, fullfilling requirement (xx). A hardware based MIPI core will be programmed into the programmable logic and will handle receiving of camera footage.
+The MIPI protocol is a widely adopted protocol for image and video transmission, fullfilling requirement . A hardware based MIPI core will be programmed into the programmable logic and will handle receiving of camera footage.
```
Comment (peterdeutsch) [inline docs/design/3_1_1_video_capture.tex]: Requirement is missing here

File: docs/validation/validation_specification.tex [hunk 2]
```diff
@@ -191,7 +191,29 @@ \subsection{Computing Platform Tests}
 \subsubsection{General Tests}
 
 \subsubsection{Camera Tests}
+\begin{table}[H]
+	\begin{tabular}{ |p{1.2cm}|p{1.4cm}|p{9cm}|p{1.1cm}|p{1.7cm}|}
+		\hline
+		\textbf{VR}     & \textbf{DR}      & \textbf{Test Description}  	& \textbf{Status} & \textbf{Eff. Date} \\ \hline
+		T.CAM.1 & F.CAM.1 	& Turn the camera (connected to the FPGA) is on verify that the FPGA is receiving the correct encoded video. & PEND & \\ \hline
+		T.CAM.2 & F.CAM.2 NF.CAM.6 	& Allow the camera to operate for 2 hours and verify that the hardware memory (?) contains the captured video. & PEND & \\ \hline
```
Comment (peterdeutsch) [inline docs/validation/validation_specification.tex]:
```suggestion
		T.CAM.2 & F.CAM.2 NF.CAM.6 	& Allow the camera to operate for 2 hours and verify that the recorded video can be played back. & PEND & \\ \hline
```

File: docs/validation/validation_specification.tex [hunk 3]
```diff
@@ -191,7 +191,29 @@ \subsection{Computing Platform Tests}
...
+		T.CAM.3 & NF.CAM.1 NF.CAM.2	& Set up a controlled environment at least 30 meters long, take images of an object 10 meters tall and 10 centimeters at a wide range of distances away from the camera and visually asses the images for resolvable objects. & PEND & \\ \hline
+		T.CAM.4 & NF.CAM.3	& Ensure the specification of the camera states a capture rate of more than 10 frames per second. & PEND & \\ \hline % remove as needed
```
Comment (peterdeutsch) [inline docs/validation/validation_specification.tex]: Not sure if this counts as a test :)

File: docs/validation/validation_specification.tex [hunk 4]
```diff
@@ -191,7 +191,29 @@ \subsection{Computing Platform Tests}
...
+\end{table}
 \subsubsection{Machine Learning Tests}
+\begin{table}[H]
+	\begin{tabular}{ |p{1.1cm}|p{1.3cm}|p{9cm}|p{1.1cm}|p{1.7cm}|}
+		\hline
+		\textbf{VR}     & \textbf{DR}      & \textbf{Test Description}  	& \textbf{Status} & \textbf{Eff. Date} \\ \hline
+		T.ML.1 & F.ML.1		& Ensure the machine learning model is implemented on the programmable logic of the FPGA. & PEND & \\ \hline % removable
+		T.ML.2 & F.ML.2		& Ensure the machine learning model can process video & PEND & \\ \hline % removable
+		T.ML.3 & F.ML.3 NF.ML.1 NF.ML.2	& Feed live encoded video to the machine learning algorithm on the FPGA and verify that results are outputted in real time and expected, the execution time (video input to metadata output) is one second or less and at least 2 frames are computed each second & PEND & \\ \hline
+		T.ML.4 & F.ML.4		& Feed live encoded video and pre-recorded video in two separate cases into the machine learning algorithm on the FPGA and verify that outputted results are expected & PEND & \\ \hline
```
Comment (peterdeutsch) [inline docs/validation/validation_specification.tex]: In this one, are you saying that we should do it once live, and once with the recording, and see if they're identical? It's not super clear.

File: docs/design/3_1_1_video_capture.tex (hunk 1 above)
Comment (Arthur-H21) [reply docs/design/3_1_1_video_capture.tex]: hmm my change didn't get committed

**Pull request closed without merging**